Fix CI

There's a few issues with the CI:

1. [ ] It throws trivial warnings like `-Wparantheses`

	```
	lib/src/TheThingsNetwork.cpp: In member function 'void TheThingsNetwork::configureUS915(uint8_t, uint8_t)':
	lib/src/TheThingsNetwork.cpp:422:37: warning: suggest parentheses around '&&' within '||' [-Wparentheses]
	if (ch == ch500 || ch <= chHigh && ch >= chLow) {
	^
	```

	I couldn't find out how to disable a warning:
	https://github.com/platformio/platformio/issues/816

2. [ ] It throws warnings for missing builtin libraries like Wire:

	```
	In file included from lib/src/Hackscribble_MCP9804.cpp:24:0:
	lib/src/Hackscribble_MCP9804.h:27:18: fatal error: Wire.h: No such file or directory
	#include <Wire.h>
	^
	compilation terminated.
	```

3. [ ] When I add ` --project-option="lib_deps=Wire"` to fix it fails over errors in libs we include:

	```
	lib/src/Hackscribble_MCP9804.cpp:246:118: error: default argument given for parameter 2 of 'void Hackscribble_MCP9804::configureAlert(boolean, uint16_t)' [-fpermissive]
	void Hackscribble_MCP9804::configureAlert(boolean action, uint16_t settings = ALERT_ALL | ALERT_LOW | ALERT_INTERRUPT)
	^
	In file included from lib/src/Hackscribble_MCP9804.cpp:24:0:
	lib/src/Hackscribble_MCP9804.h:99:8: error: after previous specification in 'void Hackscribble_MCP9804::configureAlert(boolean, uint16_t)' [-fpermissive]
	void configureAlert(boolean action, uint16_t settings = ALERT_ALL | ALERT_LOW | ALERT_INTERRUPT);
	^
	lib/src/Hackscribble_MCP9804.cpp: In member function 'uint8_t Hackscribble_MCP9804::_readRegister8(MCP9804_Register)':
	lib/src/Hackscribble_MCP9804.cpp:51:1: warning: control reaches end of non-void function [-Wreturn-type]
	}
	^
	lib/src/Hackscribble_MCP9804.cpp: In member function 'uint16_t Hackscribble_MCP9804::_readRegister16(MCP9804_Register)':
	lib/src/Hackscribble_MCP9804.cpp:79:1: warning: control reaches end of non-void function [-Wreturn-type]
	}
	^
	*** [.pioenvs/sparkfun_promicro8/lib/src/Hackscribble_MCP9804.o] Error 1
	```

	I'm not sure how we can ignore these files to be compiled/checked.

--- a/.travis.yml
+++ b/.travis.yml
@@ -14,5 +14,6 @@ env:
 install:
     - pip install -U platformio
 
-#script:
+script:
 #    - platformio ci --lib=src --board=leonardo --board=sparkfun_promicro8
+    - "echo \"Warning: no test specified\" && exit 0"

--- a/.travis.yml
+++ b/.travis.yml
@@ -14,5 +14,5 @@ env:
 install:
     - pip install -U platformio
 
-script:
-    - platformio ci --lib=src --board=leonardo --board=sparkfun_promicro8
+#script:
+#    - platformio ci --lib=src --board=leonardo --board=sparkfun_promicro8

--- a/src/TheThingsNetwork.cpp
+++ b/src/TheThingsNetwork.cpp
@@ -419,7 +419,7 @@ void TheThingsNetwork::configureUS915(uint8_t sf, uint8_t fsb) {
     str = "";
     str.concat(F("mac set ch status "));
     str.concat(ch);
-    if (ch == ch500 || (ch <= chHigh && ch >= chLow)) {
+    if (ch == ch500 || ch <= chHigh && ch >= chLow) {
       str.concat(F(" on"));
       sendCommand(str);
       if (ch < 63) {

